src/components/appointments/types.ts

```diff
@@ -0,0 +1,39 @@
+import type {AppointmentProduct} from '@/data/appointments';
+
+export type AppoinmentStep = 'producten' | 'kalender' | 'contactgegevens';
```

Comment:
Would it not be better to use:
```suggestion
export type AppoinmentStep = keyof AppointmentDataByStep;
```
This way, the `AppoinmentStep` and `AppointmentDataByStep` will always be in sync.